### ISSUE TITLE
数据预处理的代码会开吗

### ISSUE THREAD

**Issue by liwang0621** [Francis-Rings/StableAvatar#8]
比如说，mask mask提取的，现在好像只有lip mask的，还有不同分辨率视频的处理逻辑

**Comment by Francis-Rings** [comment]
Hi, for the details of human mask extraction, please refer to [StableAnimator repo](https://github.com/Francis-Rings/StableAnimator). The Human Face Mask Extraction section in the tutorial provides off-the-shelf codes. For the mixed resolution training setting, please refer to train_1B_rec_vec.py (480x832 and 832x480). For the multi-resolution inference, the preprocessing code for inference on videos of different resolutions is currently under review and will be open-sourced later.

**Comment by Francis-Rings** [comment]
We will release the 14B model in the future. However, the Wan2.1-14B backbone significantly increases the inference latency and GPU memory consumption during training and inference. It indicates limited efficiency in terms of performance-to-resource ratio. Our goal is to explore audio-driven, infinite-length avatar video generation. Therefore, any backbone (Wan2.2, LTX, HunyuanVideo) can be adapted to our method and fine-tuned on a video dataset to enable infinite-length avatar video generation.
